### ISSUE TITLE
Stacked Bars 100

### ISSUE THREAD

**Issue by marensius** [apexcharts/Blazor-ApexCharts#30]
Would it be possible to add 100% Stacked Bar?
Also, is it possible to flip the Basic Charts - Bar Horizontally and Vertically as one would like?

Ref https://apexcharts.com/javascript-chart-demos/bar-charts/stacked-100/

**Comment by joadan** [comment]
Hello,

You should be able to use the options, something like this:


```
        options.Chart = new Chart
            {
                Type = ChartType.Bar,
                Stacked = true,
                StackType = StackType.Percent100
            };

        options.PlotOptions = new PlotOptions
            {
                Bar = new PlotOptionsBar
                {
                    Horizontal = true
                }
            };
```

**Comment by joadan** [comment]
Added Sample [https://apexcharts.github.io/Blazor-ApexCharts/bar-column-chart](https://apexcharts.github.io/Blazor-ApexCharts/bar-column-chart)

**Comment by marensius** [comment]
Love it!

Is there somewhere I'm missing how I could be able to read up on how to use f.ex options.Colors ?

**Comment by joadan** [comment]
Have you checked the apex chart [documentation](https://apexcharts.com/docs/creating-first-javascript-chart/)? 

I need to understand it better myself in order to make it easier to use from BlazorApexCharts, but for now you should  be able to use the options directly. 

Please let me know if you have ideas on how we can improve.

Thanks

**Comment by marensius** [comment]
Yeah checked it out, started to give me a headache and the solution was right in front of me.

Could you add this to your example?

**options.Colors = new List<string> { "#d9534f", "#f0ad4e"};**

The project is great, keep up the good work

**Comment by marensius** [comment]
Ah, it was just ment to help out others.
I was trying to controll the colors of the bar and that options helped out =) 

The ApexChart changes the color based on how much data is inserted(to my understanding ofc)
Your example code helped out with that, it even works with ChartType.Donut :)

```
	private ApexChartOptions<Order> GetChartOptions()
	{
		var options = new ApexChartOptions<Order>();
		options.Colors = new List<string> { "#5cb85c", "#f0ad4e", "#d9534f" };
		return options;
	}
```

![image](https://user-images.githubusercontent.com/9416136/136103574-5d3026ea-2041-453b-b420-c4cf437266ea.png)

**Comment by joadan** [comment]
Sorry, I missread you comment.
I'll update the sample..

I'm also looking into  if we can provide the color in the Chart/Series to make it easier to use. 
Check out the Sample [https://apexcharts.github.io/Blazor-ApexCharts/series-options](https://apexcharts.github.io/Blazor-ApexCharts/series-options).

This one is actually also set the color array, this is probably not correct. But the general idea would be the same.
However this would not work for Pie/Dounut charts as it stands.
